requirements-dev.txt

```diff
@@ -13,3 +13,4 @@ typing-extensions<=4.5.0; python_version < '3.8'
 packaging
 build
 prompt_toolkit
+ruff
```

Comment:
This isn't necessary - the second line in this file installs everything from `requirements-tests.txt`, which already contains `ruff`